app/views/applications/show.html.erb

```diff
@@ -43,4 +43,8 @@
       </a>
     </div>
     <% end %>
+
+    <footer class="card-footer">
+      <%= link_to "approve all", "/pets/#{@applicant.id}/approval?applicant=#{@applicant.id}&approve_all=true", method: :patch, class: "card-footer-item is-primary" %>
```

Comment:
Add query params to send the approval all to use the  update action inthe pet_applications controller

Reply:
Clever! Query params are still hard for me, but this use case makes total sense 👍